### ISSUE TITLE
V1.6.5 FFmpegFrameGrabber.grab not grabbing all frames

### ISSUE THREAD

**Issue by luunn54** [bytedeco/javacv#1678]
ver 1.5.6

Issue: FFmpegFrameGrabber.grab not grabbing all frames
Test code:
```
        try {
            FFmpegLogCallback.set();

            FFmpegFrameGrabber g = new FFmpegFrameGrabber("st.mp4");
            g.start();

            int count_frame = 0;
            while (true) {
                Frame frame = g.grab();
                if (frame == null) {
                    break;
                }

                count_frame++;
            }

            g.stop();
            System.out.println("Number of frames: " + count_frame);
        } catch (FFmpegFrameGrabber.Exception e) {
            e.printStackTrace();
        }
```

Output:
```
RealSense devices found: 0
RealSense2 devices found: 0
(1) Could not find the CLEyeMulticam.dll
(2) Could not find the CLEyeMulticam.dll
Info: Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'st.mp4':

Info:   Metadata:

Info:     major_brand     : 
Info: isom
Info: 

Info:     minor_version   : 
Info: 512
Info: 

Info:     compatible_brands: 
Info: isomiso2avc1mp41
Info: 

Info:     encoder         : 
Info: Lavf56.25.101
Info: 

Info:   Duration: 
Info: 00:00:08.02
Info: , start: 
Info: 0.000000
Info: , bitrate: 
Info: 251 kb/s
Info: 

Info:   Stream #0:0
Info: (und)
Info: : Video: h264 (High) (avc1 / 0x31637661), yuv420p, 640x360 [SAR 1:1 DAR 16:9], 117 kb/s
Info: , 
Info: 15 fps, 
Info: 15 tbr, 
Info: 15360 tbn, 
Info: 30 tbc
Info:  (default)
Info: 

Info:     Metadata:

Info:       handler_name    : 
Info: VideoHandler
Info: 

Info:       vendor_id       : 
Info: [0][0][0][0]
Info: 

Info:   Stream #0:1
Info: (und)
Info: : Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, stereo, fltp, 128 kb/s
Info:  (default)
Info: 

Info:     Metadata:

Info:       handler_name    : 
Info: SoundHandler
Info: 

Info:       vendor_id       : 
Info: [0][0][0][0]
Info: 

Number of frames: **449**
```

But the video has 465 frames
ffprobe -i /Users/lunguyen/test-ffmpeg/st.mp4  -show_frames 2>&1 | grep -c media_type=audio
345
ffprobe -i /Users/lunguyen/test-ffmpeg/st.mp4  -show_frames 2>&1 | grep -c media_type=video
120

OS: Mac and Centos
Version: 1.5.6-snapshot

org.bytedeco
javacv-platform
1.5.6-SNAPSHOT

Test video:
[st.mp4.zip](https://github.com/bytedeco/javacv/files/6912952/st.mp4.zip)

**Comment by saudet** [comment]
Is it missing audio frames or video frames?

**Comment by luunn54** [comment]
Hi @saudet, it's missing some video frames

**Comment by saudet** [comment]
I'm getting 456 frames with JavaCV 1.5.6-SNAPSHOT:
```
Number of frames: 456
```
Does this happen only on Windows??

**Comment by saudet** [comment]
Ah, no, the total number of frames is 465. I fixed this issue in commit https://github.com/bytedeco/javacv/commit/bf836e03ce324a4a902077690c6393f70601d902, which is also somewhat related to issue #1677. It looks like the old API would ignore all the errors currently returned by `avcodec_send_packet()` (at least in the case of video codecs), so I've simply removed the check for those, as suggested by @DesenYang, which seems to work OK. Thanks for reporting!
